### ISSUE TITLE
Cross api query meet error

### ISSUE THREAD

**Issue by javacode123** [wundergraph/wundergraph#491]
### Bug description

1: api config:
```
const countries = introspect.graphql({
	apiNamespace: 'countries',
	url: 'https://countries.trevorblades.com/',
});

const todo = introspect.mysql({
	apiNamespace: 'todo',
	databaseURL: 'mysql://root:zjl@localhost:3306/fireboom-issue',
});

// configureWunderGraph emits the configuration
configureWunderGraphApplication({
	apis: [
		countries,
		todo,
		/*federatedApi,
        openAPI,
        graphQLAPI*/
	],
	server,
	operations,
	codeGenerators: [
		{
			templates: [
				// use all the typescript react templates to generate a client
				...templates.typescript.all,
			],
			// create-react-app expects all code to be inside /src
			// path: "../frontend/src/generated",
		},
	],
	cors: {
		...cors.allowAll,
		allowedOrigins:
			process.env.NODE_ENV === 'production'
				? [
						// change this before deploying to production to the actual domain where you're deploying your app
						'http://localhost:3000',
				  ]
				: ['http://localhost:3000', new EnvironmentVariable('WG_ALLOWED_ORIGIN')],
	},
	dotGraphQLConfig: {
		hasDotWunderGraphDirectory: false,
	},
	security: {
		enableGraphQLEndpoint: process.env.NODE_ENV !== 'production' || process.env.GITPOD_WORKSPACE_ID !== undefined,
	},
});
```

2: db table:
```
CREATE TABLE `test` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(191) COLLATE utf8mb4_unicode_ci NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=3 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

3: operation:
```
query MyQuery($code: ID! @internal) {
  todo_findFirsttest {
    id
    name @export(as:"code")
    _join {
      countries_country(code: $code) {
        code
        capital
      }
    }
  }
}
```
4: http quey
![Uploading image.png…]()

### Environment & setup

- OS: 
- Go version: 
- Database: 
- Node.js version:

**Comment by javacode123** [comment]
http response:
{
    "errors": [
        {
            "message": "GraphQL operations must contain a non-empty `query` or a `persistedQuery` extension.",
            "extensions": {
                "code": "INTERNAL_SERVER_ERROR"
            }
        }
    ],
    "data": {
        "todo_findFirsttest": {
            "id": 4,
            "name": "AE",
            "_join": {
                "countries_country": null
            }
        }
    }
}

**Comment by javacode123** [comment]
table recoders:
![image](https://user-images.githubusercontent.com/11720169/209555863-5d993c33-8a5b-4c48-8baa-7ea48000d047.png)

**Comment by spetrunin** [comment]
@javacode123 this fix will be included in the next sdk release

thanks a lot for you report!

**Comment by javacode123** [comment]
@spetrunin sorry, it seems not fix, i use wundergraph code with commit `556fae1a`, it sill return err
```
{
    "errors": [
        {
            "message": "GraphQL operations must contain a non-empty `query` or a `persistedQuery` extension.",
            "extensions": {
                "code": "INTERNAL_SERVER_ERROR"
            }
        }
    ],
    "data": {
        "todo_findFirsttest": {
            "id": 4,
            "name": "AD",
            "_join": {
                "countries_country": null
            }
        }
    }
}
```

**Comment by javacode123** [comment]
or i need update node wundergraph/sdk to which version

**Comment by spetrunin** [comment]
it will be included in next release `@wundergraph/sdk@0.130.3`

**Comment by spetrunin** [comment]
`@wundergraph/sdk@0.131.0` was released with this fix
